roles/openshift_health_checker/openshift_checks/fluentd_config.py

```diff
@@ -0,0 +1,190 @@
+"""
+Module for performing checks on an Fluentd logging deployment configuration
+"""
+
+import os
+import json
+
+from openshift_checks import OpenShiftCheck, get_var, OpenShiftCheckException
+
+
+class FluentdConfig(OpenShiftCheck):
+    """Module that checks logging configuration of an integrated logging Fluentd deployment"""
+    name = "fluentd_config"
+    tags = ["health"]
+
+    logging_namespace = None
+
+    @classmethod
+    def is_active(cls, task_vars):
+        return super(FluentdConfig, cls).is_active(task_vars)
+
+    def run(self, tmp, task_vars):
+        """Check that Fluentd has running pods, and that its logging config matches Docker's logging config."""
+
+        self.logging_namespace = get_var(task_vars, "openshift_logging_namespace", default="logging")
+        config_error = self.check_logging_config(task_vars)
+        if config_error:
+            msg = ("The following Fluentd logging configuration problem was found:"
+                   "\n-------\n"
```

Comment:
Right now, we seem to use it for similarly-worded error messages in [the](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_health_checker/openshift_checks/logging/elasticsearch.py#L36) [logging](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_health_checker/openshift_checks/logging/curator.py#L33) [stack](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_health_checker/openshift_checks/logging/kibana.py#L45) [checks](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_health_checker/openshift_checks/logging/fluentd.py#L34).